packages/sanity/src/core/studio/tasks/components/TasksSidebar/TasksSidebar.tsx

```diff
@@ -0,0 +1,36 @@
+import {Card} from '@sanity/ui'
+import styled from 'styled-components'
+import {AnimatePresence, motion} from 'framer-motion'
+import {useTasksEnabled, useTasks} from '../../context'
+import {TasksSidebarHeader} from './TasksSidebarHeader'
+
+const SidebarRoot = styled(Card)`
+  height: 100%;
+  width: 360px;
+  margin-left: 4px;
+  box-shadow:
+    0px 6px 8px -4px rgba(134, 144, 160, 0.2),
+    0px -6px 8px -4px rgba(134, 144, 160, 0.2);
+`
+
+export function TasksStudioSidebar() {
+  const {enabled} = useTasksEnabled()
+  const {isSidebarOpen} = useTasks()
+
+  if (!enabled) return null
+  return (
+    <AnimatePresence initial={false}>
+      {isSidebarOpen && (
+        <motion.div
+          initial={{translateX: 16, opacity: 0}}
```

Comment:
Since these are static values – can we move them outside the component scope? 

```tsx
import {Variants, Transition} from 'framer-motion'

const VARIANTS: Variants = {
  hidden: {opacity: 0, x: 16},
  visible: {opacity: 1, x: 0},
}

const TRANSITION: Transition = {duration: 0.2}

return (
  <motion.div variants={VARIANTS} transition={TRANSITION} initial="hidden" animate="visible">
    ...
  </motion.div>
)
```